src/components/Header/components/NavigationList/NavigationList.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Box, Button } from '@mui/material';
+import { pages } from '../../constants/pages';
+import './NavigationList.scss';
+
+export const NavigationList = () => (
+  <Box className="header-navigation" component="nav">
+    {pages.map(({ name, url }, index) => (
```

Comment:
types should be provided

Reply:
1. created interface Page in pages.ts
2. added type for index